doc/releases/releases.yml

```diff
@@ -15,6 +15,10 @@ releases:
   octopus:
     target_eol: 2022-06-01
     releases:
+      - version: 15.2.7
+        released: TBD
```

Comment:
I believe the Jenkins docs build failure is caused by this line, which will need to be updated before this PR merges anyway.